<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DefaultValuesWidget</class>
 <widget class="QWidget" name="DefaultValuesWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>772</width>
    <height>595</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>变量默认值写入</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>3</number>
   </property>
   <property name="leftMargin">
    <number>3</number>
   </property>
   <property name="topMargin">
    <number>3</number>
   </property>
   <property name="rightMargin">
    <number>3</number>
   </property>
   <property name="bottomMargin">
    <number>3</number>
   </property>
   <item>
    <widget class="QTableView" name="tableView_default_values">
     <property name="minimumSize">
      <size>
       <width>600</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="focusPolicy">
      <enum>Qt::FocusPolicy::NoFocus</enum>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SelectionMode::SingleSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectionBehavior::SelectRows</enum>
     </property>
     <property name="showGrid">
      <bool>false</bool>
     </property>
     <attribute name="horizontalHeaderHighlightSections">
      <bool>false</bool>
     </attribute>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <attribute name="verticalHeaderHighlightSections">
      <bool>false</bool>
     </attribute>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_8">
     <property name="spacing">
      <number>5</number>
     </property>
     <item>
      <widget class="QLabel" name="label_status">
       <property name="minimumSize">
        <size>
         <width>110</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>110</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>成功/失败</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBox_enable_all_items">
       <property name="text">
        <string>启用全部</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbtn_reset">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>26</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>80</width>
         <height>26</height>
        </size>
       </property>
       <property name="text">
        <string>复位</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbtn_write">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>26</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>80</width>
         <height>26</height>
        </size>
       </property>
       <property name="text">
        <string>写入默认值</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbtn_write_all">
       <property name="minimumSize">
        <size>
         <width>140</width>
         <height>26</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>140</width>
         <height>26</height>
        </size>
       </property>
       <property name="text">
        <string>一键写入全部默认值</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
